presto-main/src/main/java/com/facebook/presto/operator/HttpPageBufferClient.java

```diff
@@ -598,7 +598,7 @@ public PagesResponse handle(Request request, Response response)
                 }
             }
             catch (PageTransportErrorException e) {
-                throw new PageTransportErrorException(format("Error fetching %s: %s", request.getUri().toASCIIString(), e.getMessage()), e);
+                throw new PageTransportErrorException("Error fetching %s " + request.getUri().toASCIIString(), e);
```

Comment:
I guess you either want to remove `%s` or keep `format`.

Reply:
Yup. My bad :)